src/slacko.mli

```diff
@@ -149,6 +149,14 @@ type message_length_error = [
   | `Msg_too_long
 ]
 
+(** When posting a message with attachments, you may receive this error if you
```

Comment:
I am a bit confused by this docstring, because it seems to state:

 * The maximum is 100 attachments
 * The maximum is 20 attachments